src/libraries/Microsoft.Extensions.Logging.Abstractions/tests/Microsoft.Extensions.Logging.Generators.Tests/LoggerMessageGeneratorParserTests.cs

```diff
@@ -341,22 +341,6 @@ partial class C
             Assert.Single(diagnostics);
             Assert.Equal(DiagnosticDescriptors.InconsistentTemplateCasing.Id, diagnostics[0].Id);
         }
```

Comment:
I think we should delete this case too

Reply:
Can I remove it in a separated PR? Just because we have to clean the diagnostic descriptor (that was already merged), and the markdown that contains all other diagnostics.